src/handlers/graphql.ts

```diff
@@ -31,5 +33,5 @@ export const apolloServer = new ApolloServer<MyContext>({
 });
 
 export const server = startServerAndCreateLambdaHandler<MyContext>(apolloServer, {
-	context: async () => ({ dataSources: { technologies: [] } }),
+	context: async () => ({ dataSources: { technologies: [], genres: [] } }),
```

Comment:
I'd remove the references to technologies at this point.

Reply:
@jesus4497 this is why I made that comment in your former PR btw. I knew someone would forget.

Reply:
technology initial example removed from repo